test/corpus/expressions.txt

```diff
@@ -2914,6 +2921,9 @@ ary[*{0}, **{count: 1}]
 
 a.[count: 1, values: [5]] =0
 b.["q": 1, q?: 3, %q(stuff): [5]] =0
+
+a.[2] %= 5
```

Comment:
Does this supported chained version of this? (`a.[1].[2].method`)

Reply:
Great catch, it should but I'll add test cases